src/components/TruncatedQuery/TruncatedQuery.tsx

```diff
@@ -36,8 +20,11 @@ interface OneLineQueryWithPopoverProps {
 
 export const OneLineQueryWithPopover = ({value = ''}: OneLineQueryWithPopoverProps) => {
     return (
-        <CellWithPopover contentClassName={b('popover-content')} content={value}>
-            {value}
+        <CellWithPopover
```

Comment:
It seems that popover is now redundant, because we don't truncate query any more.